src/client/pfs.go

```diff
@@ -1021,6 +1061,20 @@ func (c APIClient) GetFile(repoName string, commitID string, path string, offset
 	return nil
 }
 
+func (c APIClient) GetFileStream(repoName string, commitID string, path string, offset int64, size int64, writer io.Writer) error {
+	//TODO(kdelga): understand what this limiter stuff is doing and if it's needed.
+	//if c.limiter != nil {
```

Comment:
Also not clear on what this limiter stuff is doing and whether it's needed since it works without it.

Reply:
`limiter` is used to limit the number of outstanding requests you have so you don't overwhelm the backing services. Things should work without it until you do really high stress tests.